src/main/java/com/xxxx/example/Servlet3.java

```diff
@@ -18,14 +19,22 @@ public class Servlet3 extends HttpServlet {
     protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
         //接收前端字符
         req.setCharacterEncoding("UTF-8");
+        resp.setCharacterEncoding("UTF-8");
+
         String get = req.getParameter("text");
         String hydragel = req.getParameter("time");
+        String userName = req.getParameter("userName");
         System.out.println("接收前端信息: "+ get);
         System.out.println("接收前端时间:"+ hydragel);
+//        Time hello = new Time(2,1,1);
+//        hello.toLocalTime();
+        Date hello = new Date();
+        System.out.println(hello.getTime());
+
 
         //上传到数据库 commit前请不要暴露地址
-        Database myDatabase = new Database("<connection string goes here>", "DatingAppStaging");
-        myDatabase.insert_post("tianxianbaobao", get);
+        Database myDatabase = new Database("", "DatingAppStaging");
```

Comment:
We are re-creating a lot of database connections across multiple file, the database connection object should be re-used

Reply:
Example of a piece of code doing something similar to what we want.
https://gist.github.com/leefsmp/b4c089734852c793cf85

Reply:
That is what I am struggling with right now. Thanks for this document. Although some code is beyond my stage, I will try to understand this.